Trimble.Modus.Components/Controls/TMSegmentedControl/TMSegmentedItem.xaml

```diff
@@ -0,0 +1,72 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Border xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
Frame causes a unwanted shadow in iOS. And we shouldn't be using Frame in MAUI.
I have replaced Frame with Border and I have run the app and verified it. Please run and check.